gltf/gltfpack.cpp

```diff
@@ -405,6 +406,18 @@ static void process(cgltf_data* data, const char* input_path, const char* output
 	size_t mesh_offset = 0;
 	size_t material_offset = 0;
 
+	for (size_t i = 0u; i < data->samplers_count; ++i)
+	{
+		const cgltf_sampler& sampler = data->samplers[i];
+
+		comma(json_samplers);
+		append(json_samplers, "{");
+		writeSampler(json_samplers, sampler);
+		if (settings.keep_extras)
```

Comment:
To keep extras for a given element, additional code is necessary during parsing. I would remove this for now, as gltfpack only adds support for extras with a solid use case and here it's not clear what the usecase is.

Reply:
Sure. Removed extras for now.